doc/strategies/RFQv2.md

```diff
@@ -0,0 +1,44 @@
+# RFQv2
+
+The RFQv2 is a contract that settles a trade between two parties. Normally it requires an off-chain quoting system which provides quotes and signatures from certain makers. A user may request for a quote of a specific trading pair and size. Upon receiving the request, any maker willing to trade can respond with a quote to the user. If the user accepts it, then both parties will sign the trading details and submit the order with signatures to the RFQv2 contract. After verifying signatures, the RFQv2 contract will transfer tokens between user and maker accordingly.
+
+## Token allowance
+
+The main difference between RFQv2 and previous RFQ is how token allownace is managed. The RFQv2 now supports multiple forms of token allownace in order to cover different security assumptions at the same time. A RFQv2 trade should be submitted with sufficient allownace info to specify how tokens should be transfered between two sides.
+
+### List of supported allowance form
+
+-   Approval on Tokenlon's Allowance Target
+-   Approval on RFQv2 contract
+-   Approval on Uniswap Permit2
+-   Native token permit
+
+## Offer Format
+
+| Field               |  Type   | Description                                                                        |
+| ------------------- | :-----: | ---------------------------------------------------------------------------------- |
+| taker               | address | The address of the taker of an offer. An offer can be filled only by this address. |
+| maker               | address | The address of the maker of an offer.                                              |
+| takerToken          | address | The address of taker token.                                                        |
+| takerTokenAmount    | uint256 | The amount of taker token.                                                         |
+| makerToken          | address | The address of maker token.                                                        |
+| makerTokenAmount    | uint256 | The amount of maker token.                                                         |
+| minMakerTokenAmount | uint256 | The minimum amount of maker token. Should be identical to makerTokenAmount here.   |
+| expiry              | uint256 | The timestamp of the expiry.                                                       |
+| salt                | uint256 | A random number included in an offer to avoid replay attack.                       |
+
+## Signature and Relayer
```

Comment:
Relayer is not mentioned in this section?

Reply:
#191